client/my-sites/plugins/plugin-ratings/index.jsx

```diff
@@ -43,23 +44,25 @@ export default React.createClass( {
 	},
 
 	renderRatingTier( ratingTier ) {
-		const numberOfRatings = ( this.props.ratings && this.props.ratings[ ratingTier ] ) ? this.props.ratings[ ratingTier ] : 0;
+		const { ratings, slug, numRatings } = this.props;
+		const numberOfRatings = ( ratings && ratings[ ratingTier ] ) ? ratings[ ratingTier ] : 0;
+
 		return (
 			<div className="plugin-ratings__rating-tier" key={ `plugins-ratings__tier-${ ratingTier }` }>
 				<a className="plugin-ratings__rating-container" target="_blank" rel="noopener noreferrer"
-					onClick={ analytics.ga.recordEvent( 'Plugins', 'Clicked Plugin Ratings Link', 'Plugin Name', this.props.slug ) }
+					onClick={ analytics.ga.recordEvent( 'Plugins', 'Clicked Plugin Ratings Link', 'Plugin Name', slug ) }
 					href={ this.buildReviewUrl( ratingTier ) }
```

Comment:
Sure, it can. But I have something in my mind that will actually make it reusable, and will probably introduce some logic in the function. But that's for another PR. 😉